examples/vision/attention_mll_classification.py

```diff
@@ -0,0 +1,541 @@
+"""
+Title: Classification using Attention-based Deep MIL
+Author: [Mohamad Jaber](https://www.linkedin.com/in/mohamadjaber1/)
+Date created: 2021/08/16
+Last modified: 2021/08/16
+Description: A neural network model that uses attention weights on instance-level to decide the output class.
+"""
+"""
+## Introduction
+
+Instead of having individual labelled instances, this neural network model receives a single label for each bag of instances.
+The bag is labelled positive if it contains atleast one positive instance and negative if it does not contain any.
+The MIL problem is stated as the learning Bernoulli distribution of the bag label where output probability is parametermized by the algorithm.
+The attention mechanism provides insight into the contribution of each instance to the bag label.
+
+Besides the implementation of this mechanism, some **regularization** techniques, **ensemble averaging** (for stability)
+and dealing **imbalanced data** will be covered.
+
+## References
+
+- The paper [Attention-based Deep Multiple Instance Learning](https://arxiv.org/pdf/1802.04712.pdf) is authored by Maximilian Ilse,
+Jakub M. Tomczak and Max Welling.
+- Some of attention operator code implementation was inspired from https://github.com/utayao/Atten_Deep_MIL.
+- Imbalanced data [tutorial](https://www.tensorflow.org/tutorials/structured_data/imbalanced_data) by TensorFlow.
+"""
+"""
+## Setup
+"""
+
+import numpy as np
+import os
+from tqdm import tqdm
+import tempfile
+from tensorflow.python.keras.datasets import mnist, fashion_mnist
+from tensorflow.python.keras.models import Model
+from tensorflow.python.keras import layers
+from tensorflow.python.keras import callbacks
+from tensorflow.python.keras import backend as K
+from tensorflow.python.keras import initializers, regularizers
+from matplotlib import pyplot as plt
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+plt.style.use("ggplot")
+
+"""
+## Create Dataset
+
+In this example, either of `mnist` or `fashion_mnist` data can be used.
+At first we will set up the configurations, create the function for preparing them and then load the datasets that will be *used*.
```

Comment:
In general, the text copy of this example will need extensive rework. Do you know someone with tech writing experience you could collaborate with or who could proofread and edit the text?

Reply:
I have checked other Keras examples text and have now updated the text in this example. Thanks to @ksalama, I asked if he could proofread it and he also gave me some insights. 
I have tried to make clearer now. Please let me know if it needs further update.